tdrs-backend/tdpservice/parsers/schema_defs/tanf/t6.py

```diff
@@ -0,0 +1,168 @@
+        Field(item="6A", name='NUM_DENIED', type='number', startIndex=55, endIndex=63,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="7A", name='ASSISTANCE', type='number', startIndex=79, endIndex=91,
+              required=True, validators=[validators.isInLimits(0, 999999999999)]),
+        Field(item="8A", name='NUM_FAMILIES', type='number', startIndex=115, endIndex=123,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="9A", name='NUM_2_PARENTS', type='number', startIndex=139, endIndex=147,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="10A", name='NUM_1_PARENTS', type='number', startIndex=163, endIndex=171,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="11A", name='NUM_NO_PARENTS', type='number', startIndex=187, endIndex=195,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="12A", name='NUM_RECIPIENTS', type='number', startIndex=211, endIndex=219,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="13A", name='NUM_ADULT_RECIPIENTS', type='number', startIndex=235, endIndex=243,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="14A", name='NUM_CHILD_RECIPIENTS', type='number', startIndex=259, endIndex=267,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="15A", name='NUM_NONCUSTODIALS', type='number', startIndex=283, endIndex=291,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="16A", name='NUM_BIRTHS', type='number', startIndex=307, endIndex=315,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="17A", name='NUM_OUTWEDLOCK_BIRTHS', type='number', startIndex=331, endIndex=339,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+        Field(item="18A", name='NUM_CLOSED_CASES', type='number', startIndex=355, endIndex=363,
+              required=True, validators=[validators.isInLimits(0, 99999999)]),
+    ],
+)
+
+s2 = RowSchema(
+    model=TANF_T6,
+    preparsing_validators=[
+        validators.hasLength(379),
+    ],
+    postparsing_validators=[
+        validators.sumIsEqual("NUM_APPLICATIONS", ["NUM_APPROVED", "NUM_DENIED"]),
+        validators.sumIsEqual("NUM_FAMILIES", ["NUM_2_PARENTS", "NUM_1_PARENTS", "NUM_NO_PARENTS"]),
+        validators.sumIsEqual("NUM_RECIPIENTS", ["NUM_ADULT_RECIPIENTS", "NUM_CHILD_RECIPIENTS"]),
+    ],
+    fields=[
+        Field(item="0", name='RecordType', type='string', startIndex=0, endIndex=2,
+              required=True, validators=[]),
+        Field(item="3A", name='CALENDAR_YEAR', type='number', startIndex=2, endIndex=6,
```

Comment:
@elipe17 do we need this? this is not a data element. `CALENDAR_QUARTER` has a poor label by default unfortunately, but its technically exactly the same as the label in the header and takes on value: YYYYQ. I dont feel too strongly about this;  most important is to add `RPT_MONTH_YEAR` based on YYYYQ when restructuring to month-level records.